Fix GTalk messaging tests

dborkan opened the pull request:
Fix messaging tests from GTalk (changing messageData.userId to messageData.from.userId)

ryscheng commented:
Looks good, but kinda confused. There's no way the tests should have passed without this change right? When we reviewed pull req https://github.com/freedomjs/freedom-social-xmpp/pull/85
were the tests passing?

dborkan commented:
The tests were passing, then I hit the XMPP throttle limit because I ran them too many times and they started failing.  Then I refactored and introduced this bug, and didn't realize that the tests were failing because of it when I merged instead of the XMPP throttling.  Sorry.
